evidence/pool.go

```diff
@@ -523,19 +399,26 @@ func (evpool *Pool) listEvidence(prefixKey byte, maxBytes int64) ([]types.Eviden
 		return nil, totalSize, fmt.Errorf("database error: %v", err)
 	}
 	defer iter.Close()
+	var protoOverheadBytes int64 = 3
```

Comment:
```suggestion
	const protoOverheadBytes int64 = 3
```

can we add a comment here? e.g `1 byte - field key, 1 byte - value length prefix`, etc.